integration/clean/clean_ami/clean_ami.go

```diff
@@ -1,79 +0,0 @@
-// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
For the clean AWS Resources, it would be better to keep it here since we also clean internal resources and various resources not relate  to integration test

Reply:
Hmm doesn't seem like it's used anywhere else in the code (tests ran after this deletion)? Can you show me where this is used?

Reply:
For example: [clean_ami](https://github.com/aws/amazon-cloudwatch-agent/blob/master/.github/workflows/clean-aws-resources.yml#L28), [clean_dedicated_host](https://github.com/aws/amazon-cloudwatch-agent/blob/master/.github/workflows/clean-aws-resources.yml#L46)

Reply:
oh I see, good catch! Will check if integ test needs it and make appropriate changes